spec/realtime/channel.test.js

```diff
@@ -2,7 +2,9 @@
 
 define(['ably', 'shared_helper'], function(Ably, helper) {
 	var exports = {},
-		displayError = helper.displayError;
+		displayError = helper.displayError,
```

Comment:
Don't think so..? Was two tabs before, is too tabs after. The diff was changing `;` to `,`.